### PR TITLE
AB#6554 Don't leak responses from remotes to unauthorized clients

### PR DESCRIPTION
**Pull request by lbam**
Hotfix. Will restore a sanitized version of the detailed error in a later commit.

### CODE REVIEW

File: src/dso_api/dynamic_api/remote/views.py [hunk 1]
```diff
@@ -102,11 +102,7 @@ def validate(self, serializer, raw_data):
         if not serializer.is_valid():
             raise RemoteAPIException(
                 title="Invalid remote data",
-                detail={
-                    "detail": "These schema fields did not validate:",
-                    "x-validation-errors": serializer.errors,
-                    "x-raw-response": raw_data,
-                },
+                detail="Some schema fields did not validate",
```
Comment (jjmurre) [inline src/dso_api/dynamic_api/remote/views.py:105]: We could log this information for debugging purposes. Although, when there is private info in there, that should also not end up in the logs. So, better like this.

**Review by jjmurre — APPROVED** [review]
LGTM

File: src/dso_api/dynamic_api/remote/views.py (hunk 1 above)
Comment (ngaranko) [reply src/dso_api/dynamic_api/remote/views.py:105]: Logging this information is bad indeed. But maybe logging only keys from `raw_data` to error log would be enough for debugging?

Comment (lbam) [reply src/dso_api/dynamic_api/remote/views.py:105]: Good point.

Comment (ngaranko) [reply src/dso_api/dynamic_api/remote/views.py:105]: Cool, why this was merged then?

Comment (lbam) [reply src/dso_api/dynamic_api/remote/views.py:105]: Because this is a hotfix. Next PR will have a more thorough solution.

**Pull request merged**